macosx/Controller.mm

```diff
@@ -5213,29 +5213,30 @@ - (void)setWindowSizeToFit
 
         NSDictionary* views = @{ @"scrollView" : scrollView };
 
+        CGFloat height;
         if (![self.fDefaults boolForKey:@"AutoSize"])
         {
-            //only set a minimum height constraint
-            CGFloat height = self.minScrollViewHeightAllowed;
+            // Only set a minimum height constraint
+            height = self.minScrollViewHeightAllowed;
```

Comment:
Please keep the declaration inside the block instead of outside: that has been Transmission's code style general direction to minimise the scope length where variables exist.